src/cli.ts

```diff
@@ -59,10 +62,93 @@ const noop = (): void => {
     logger.info('noop performed');
 };
 
+
+
+const syncBadPathCounts = async (): Promise<void> => {
+    await useDatabaseTransaction(async () => {
+        const topics = await CourseTopicContent.findAll(
```

Comment:
I was thinking this would search for bad problem and questions and then fetch their respective topics
But since this should only be run once I think it is fine